src/Symfony/Component/HttpFoundation/Tests/File/UploadedFileTest.php

```diff
@@ -358,13 +358,16 @@ public function testGetMaxFilesize()
     {
         $size = UploadedFile::getMaxFilesize();
 
-        $this->assertIsInt($size);
+        if ($size > \PHP_INT_MAX) {
```

Comment:
You should probably provide a test case that trigger this limit.

I'm not sure the  "then branch" is triggered...

Reply:
not really sure how to go about doing that, besides maybe using ini_set() to set the upload_max_filesize to several thousand petabytes to trigger this case on x64 based systems (that I'm assuming this test suite is running on)

Reply:
any ideas?

Reply:
AppVeyor should run a 32bit PHP build, IIRC.

Reply:
Anything I can do at this point to move this forward?

Reply:
Did you try adding a test case that would fail without the change on 32b?
Since Appveyor runs a 32b version it's possible to prevent a regression here.
Doing ini_set() in the test case might be the way to go yes.